pr-validation-original.yml

```
@@ -0,0 +1,223 @@
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Run Gitleaks
+        uses: gitleaks/gitleaks-action@v2.3.2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  encoding-validation:
+      - name: Get pnpm store directory
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)